packages/codemods/package.json

```diff
@@ -8,7 +8,11 @@
   "main": "src/index.ts",
   "module": "src/index.ts",
   "typings": "lib/index.d.ts",
+  "bin": {
+    "transform": "./bin/transform.js"
+  },
   "scripts": {
+    "build": "fluentui-scripts build",
```

Comment:
Was wondering what this line was added for? Is it needed so that "people can run the transforms on their code without copying the file from the repo, they can just install the package to run it"

Reply:
The build line? It was added so that the .ts code is transpiled into .js, which would then get published.